packages/cli-platform-android/src/config/findAndroidDir.ts

```diff
@@ -14,5 +14,12 @@ export default function findAndroidDir(root: string) {
     return 'android';
   }
 
+  if (
+    fs.existsSync(path.join(root, 'build.gradle'))
```

Comment:
Why is this needed?

Reply:
When I created a android app but not used `npx react-native` script, added a `package.json` and `yarn install` only on the android root directory, then run script `npx react-native config` can't found `project.android` info.
In the iOS project, I apply the same operations,  It's can work.
<img width="538" alt="image" src="https://user-images.githubusercontent.com/20120255/226072263-54d4d684-8d0c-4ff5-b678-999c3dc7210e.png">
<img width="474" alt="image" src="https://user-images.githubusercontent.com/20120255/226072278-410fc301-2eef-48d8-8dbf-0c47118f43aa.png">
The different is `sourceDir` property when get from default value.

Reply:
This changes is not necessary, I can create a `react-native.config.js` config fie in android root directory alos.